content/03.results.md

```diff
@@ -131,25 +131,24 @@ Indeed, we observed significant upregulation (ANOVA p < 0.01) of the KRAS signal
 #### Embryonal tumors
 **Figure {@fig:Fig2}B** shows the mutational landscape for 128 primary embryonal tumors. <!--SAMPLECOUNT-->
 The majority (N = 95) of samples were medulloblastomas that spanned the spectrum of molecular subtypes (WNT, SHH, Group3, and Group 4; see **Molecular Subtyping of CNS Tumors**), as identified by subtype-specific canonical mutations.
-We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].
+We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs, displayed as other embryonal tumors) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].
 
 #### Diffuse astrocytic and oligodendroglial tumors
-In **Figure {@fig:Fig2}C**, we show genomic alterations in 61 HGGs biopsied at diagnosis: diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (HGGs, N = 26). <!--SAMPLECOUNT-->
-The single oligodendroglioma sample (N = 1) in the OpenPBTA does not contain mutations in the listed genes and is therefore not shown in this oncoprint.
+In **Figure {@fig:Fig2}C**, we show genomic alterations in 61 HGGs biopsied at diagnosis: diffuse midline gliomas (DMGs, N = 34) and other high-grade gliomas (non-midline HGGs, N = 27). <!--SAMPLECOUNT-->
```

Comment:
The varied first sentence descriptions of figures 2A-2D reads oddly here, since they are all the same kind of figure. I might have one sentence at the top saying that all are oncoprints, followed by going straight to the descriptions of results for each one. Also, as the counts of types are in the figure legend, I don't think they need to all be duplicated in the main text here.

Reply:
I don't see ns in the figure legends...

Reply:
ahh, in the captions - yes, I can remove here then.